engine/src/main/java/org/hibernate/search/cfg/spi/SearchConfiguration.java

```diff
@@ -130,4 +131,6 @@
 	 * {@link org.hibernate.search.engine.service.spi.ServiceManager}
 	 */
 	ClassLoaderService getClassLoaderService();
+
+	ErrorHandler getErrorHandler();
```

Comment:
I _had_ in mind something like "return null unless you want to override it", but that depends on how we address your other points.

I know we don't document much, especially self-explanatory methods, but when it comes to SPI a little guidance is needed.